ci/release/update-version.sh

```diff
@@ -34,10 +36,17 @@ function sed_runner() {
     sed -i.bak ''"$1"'' $2 && rm -f ${2}.bak
 }
 
-sed_runner "s/cudf=.*/cudf=${NEXT_RAPIDS_VERSION}/g" dependencies.yaml
+DEPENDENCIES=(
+  cudf
+)
+for FILE in dependencies.yaml; do
```

Comment:
This pair of `for` loops, while not strictly necessary, is meant to align with recent changes I made in RAFT and cuML. All of RAPIDS should have a similar structure for this replacement logic.